go.mod

```diff
@@ -18,10 +18,10 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.5.0
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/aperture v0.1.18-beta
-	github.com/lightninglabs/lndclient v0.16.0-9
+	github.com/lightninglabs/lndclient v0.16.0-10
```

Comment:
Do we want aperture to 0.1.19 too?

Reply:
Ah, good point, I think I'll tag a `0.1.20-beta` of aperture after bumping the version there.

Reply:
Oh and i think we need that grpc replace too!